src/utils/fhir.js

```diff
@@ -77,24 +79,24 @@ async function getResources(server, resourceType) {
  *
  * @param {string} criteria - the criteria for the named event code
  * @param {string} url - the notification endpoint url
- * @param {string} token - access token for header
  * @param {string} id - the id to assign the Subscription
  * @param {string} subscriptionTopic - R5 backport subscription topic
  * @returns a R5 Backport Subscription
  */
-function generateSubscription(criteria, url, token, id = undefined, subscriptionTopic = undefined) {
+function generateSubscription(criteria, url, id = undefined, subscriptionTopic = undefined) {
   // Add regex to check if criteria has at least one search param since HAPI is expecting this
-  const criteriaReg = /(.+)(\?)+/;
+  const criteriaReg = /(.+)(\?.)+/;
```

Comment:
Might just be a rebase error but I removed the `.` from the regex so it only checks for the question mark.
```suggestion
  const criteriaReg = /(.+)(\?)+/;
```